synapse/http/client.py

```diff
@@ -799,6 +755,75 @@ async def get_file(
         )
 
 
+class SimpleHttpClient(BaseSynapseClient):
+    """
+    An HTTP client capable of crossing a proxy and respecting a block/allow list. This
+    does set up a HTTPConnectionPool based on a multiplier of 100 times
+    hs.config.caches.global_factor, as it is responsible for pushing to Sygnal.
```

Comment:
I think it is best to leave the *why* to the comment below in the code and just note in the docstring that it does it:

```suggestion
    An HTTP client capable of crossing a proxy and respecting a block/allow list.

    This also configures a larger / longer lasting HTTP connection pool.
```